packages/nuxt/src/app/composables/state.ts

```diff
@@ -22,6 +22,18 @@ export function useState <T> (...args: any): Ref<T> {
   }
   const key = '$s' + _key
 
+  if (process.dev) {
+    const map = new Map<string, string>()
+    const defineInit = map.get(key)
```

Comment:
minor perf point, but we can likely avoid the multiple calls to `init.toString()` below by doing it just once

```suggestion
    const defineInit = map.get(key)
    const newInit = init?.toString()
```

Reply:
Thanks. I fixed it.
https://github.com/nuxt/framework/pull/8573/commits/b55c34b7d55a88bf3e298ac8391088c56202f1ac